prisma/migrations/20210915164107_feedback/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateEnum
+CREATE TYPE "Sentiment" AS ENUM ('POSITIVE', 'NEGATIVE');
+
+-- CreateTable
+CREATE TABLE "Feedback" (
+    "id" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
For my knowledge, why would there be an updated at if there is no editing on these comments

Reply:
It probably won't be updated, but we add createdAt and updatedAt to all our models as a convention